README.md

```diff
@@ -12,6 +12,38 @@ A Transitive map can be embedded as a freestanding web element or overlaid onto
 
 Transitive is supported by the [Mobility Lab](http://mobilitylab.org/) [Transit Tech Initiative](http://mobilitylab.org/tech/transit-tech-initiative/). Read more in [this Mobility Lab article](http://mobilitylab.org/2014/04/16/the-technology-behind-a-new-kind-of-travel-planning/).
 
+## Storybook
+
+To view samples of Transitive in action, follow these steps:
```

Comment:
Would be good to also link to the GitHub pages URL once that is working properly.